common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -62,7 +62,7 @@ trait ABTestSwitches {
     "Tests an additional 'curated' onwards container below the article body that is relevant to the article's pillar.",
     owners = Seq(Owner.withGithub("rcrphillips")),
     safeState = Off,
-    sellByDate = new LocalDate(2020, 12, 10),
+    sellByDate = new LocalDate(2020, 12, 20),
```

Comment:
Best to make it midweek, I think this is Sunday.

Reply:
Classic......

Reply:
Updated